modules/core/arrow-test/src/main/kotlin/arrow/test/laws/ApplicativeErrorLaws.kt

```diff
@@ -13,6 +13,7 @@ import arrow.typeclasses.ApplicativeError
 import arrow.typeclasses.Eq
 import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
+import kotlinx.coroutines.runBlocking
 
```

Comment:
Could you please use `IO.effect { }.unsafeRunSync()` instead? :D Dogfood our own lib hahah

Reply:
Fixed that with the new commit. Good to know this tip.